profile/README.md

```diff
@@ -1,7 +1,7 @@
 - <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/dvc.svg"/> DVC: Data Version Control (ML Experiments, Pipelines, Git for data) | [code](https://github.com/iterative/dvc) | [website & docs](https://dvc.org)
 - <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/cml.svg"/> CML: Continuous Machine Learning (CI/CD for ML) | [code](https://github.com/iterative/cml) | [website & docs](https://cml.dev)
 - <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/studio.svg"/> Iterative Studio (Collaboration for ML Projects) | [website](https://studio.iterative.ai)
-- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/mlem.svg"/> MLEM (Version & Deploy ML models following GitOps principles) | [code](https://github.com/iterative/mlem) | [website & docs](https://mlem.ai)
+- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/mlem.svg"/> MLEM (ML model deployment) | [code](https://github.com/iterative/mlem) | [website & docs](https://mlem.ai)
```

Comment:
hmm, sounds like Seldon or something?

Reply:
I like the website, but it's a bit too long there and has extra explanations

may be this is indeed the best

anyways, looks like it should be "ML Model Deployment" (capital first letters)

Reply:
```suggestion
- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/mlem.svg"/> MLEM (ML Model Deployment) | [code](https://github.com/iterative/mlem) | [website & docs](https://mlem.ai)
```